liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy

```diff
@@ -0,0 +1,134 @@
+package com.liferay.blade.tests
+
+import org.gradle.testkit.runner.GradleRunner
+import static org.gradle.testkit.runner.TaskOutcome.*
+import spock.lang.Specification
+import de.undercouch.gradle.tasks.download.Download;
+import java.util.jar.JarInputStream;
+import java.io.FileNotFoundException;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Request.Builder;
+
+class BladeTests extends Specification {
+	public void checkManifest(arg1='', arg2='', arg3='') {
+		def jarInputStream = new JarInputStream(new FileInputStream("${arg1}"))
+
+		jarInputStream.withCloseable {
+			if (jarInputStream.manifest.getMainAttributes().getValue("${arg2}") == "${arg3}" ) {
+				return true;
+			}
+			else {
+				return false;
+			}
+		}
+	}
+
+	public void executeBlade(arg1='', arg2='', arg3='', arg4='') {
+		"java -jar build/${bladeclijar} ${arg1} ${arg2} ${arg3} ${arg4}".execute()
+	}
+
+	def setup () {
+		def initBundle = GradleRunner.create()
+			.withProjectDir(new File(projectDir))
+			.withArguments('initBundle')
+			.build()
+
+		def bladeVersion = '1.0.1.201606142156'
+		def bladeclijar = "com.liferay.blade.cli_${bladeVersion}.jar"
+		def url = "http://releases.liferay.com/tools/blade-cli/${bladeVersion}/plugins/${bladeclijar}"
```

Comment:
Instead of downloading this via URL, we need to download this using OSGi repository client.  this way we can always get the "latest" version and and this test script wont have to hardcode the version.

See the code here for how to do that:

https://github.com/liferay/liferay-ide/blob/master/tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/BladeCLI.java#L266-L278